.github/workflows/ci2.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        distro: [ubuntu.20.04, centos.8, leap.15]
+        distro: [ubuntu.20.04, leap.15]
```

Comment:
Doesn't changing the `BASE_DISTRO` in `utils/docker/Dockerfile.centos.8` mean we can leave `centos.8` in this list?  or better yet, rename it and the Dockerfile `rocky.8`?

Reply:
Yes, we could now.  See my other comment though.